.idea/codeStyles/Project.xml

```diff
@@ -157,4 +162,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Please revert these changes. This file should not be modified. This must have been done automatically from android studio.